### DISCUSSION TITLE
Could anyone share a latest compiled installer of Sumatra?

### DISCUSSION THREAD

**Discussion by yujiangsenior** [sumatrapdfreader/sumatrapdf#2519] [Q&A] [answered]
Hi there, I downloaded the source code package, and used Visual Studio 2022 to compile it. However there're lots of errors and failed several time. Could anyone build an installation file to be used in Win10?

**Comment by GitHubRulesOK** [comment, marked as answer]
they should be on the web page ***NOTE should now be out-of-date*** 
@kjk looks like 64 bit install (https://www.sumatrapdfreader.org/dl/prerel/14506/SumatraPDF-prerel-64-install.exe) failed to upload or go AWOL, <strike> latest from page link was https://files.sumatrapdfreader.org/file/kjk-files/software/sumatrapdf/prerel/14505/SumatraPDF-prerel-64-install.exe</strike>

My download script sees https://files.sumatrapdfreader.org/file/kjk-files/software/sumatrapdf/prerel/14506/SumatraPDF-prerel-install.exe as a bad link `{
  "code": "not_found",
  "message": "File with such name does not exist.",
  "status": 404
}` but the others are good so is there another reason ?

**Reply by yujiangsenior** [reply]
Thank you so much. 
This one [https://files.sumatrapdfreader.org/file/kjk-files/software/sumatrapdf/prerel/14505/SumatraPDF-prerel-64-install.exe](https://files.sumatrapdfreader.org/file/kjk-files/software/sumatrapdf/prerel/14505/SumatraPDF-prerel-64-install.exe) stands for the 64bit version, and it's good for me as it has underline mark for short as 'u'. Cheers
